internal/backends/qt/qt_widgets/slider.rs

```diff
@@ -156,7 +156,11 @@ impl Item for NativeSlider {
                 data.pressed = 0;
                 InputEventResult::EventIgnored
             }
-            MouseEvent::Pressed { position: pos, button: PointerEventButton::Left } => {
+            MouseEvent::Pressed {
+                position: pos,
+                button: PointerEventButton::Left,
+                click_count: 0,
```

Comment:
Why limit to single click?
```suggestion
                click_count: _,
```